packages/api-server/src/bin.ts

```diff
@@ -0,0 +1,52 @@
+import path from 'path'
+
+import { config } from 'dotenv-defaults'
+import { hideBin } from 'yargs/helpers'
+import yargs from 'yargs/yargs'
+
+import { getPaths } from '@redwoodjs/project-config'
+import * as webServerCLIConfig from '@redwoodjs/web-server'
+
+import { handler as apiHandler } from './apiCLIConfigHandler'
+import { handler as bothHandler } from './bothCLIConfigHandler'
+import { apiServerCLIConfig, bothServerCLIConfig } from './cliConfig'
+
+if (!process.env.REDWOOD_ENV_FILES_LOADED) {
+  config({
+    path: path.join(getPaths().base, '.env'),
+    defaults: path.join(getPaths().base, '.env.defaults'),
+    multiline: true,
+  })
+
+  process.env.REDWOOD_ENV_FILES_LOADED = 'true'
+}
+
+process.env.NODE_ENV ??= 'production'
+
+yargs(hideBin(process.argv))
+  .scriptName('rw-server')
+  .strict()
+  .alias('h', 'help')
+  .alias('v', 'version')
+  .command(
+    '$0',
+    bothServerCLIConfig.description,
+    // @ts-expect-error The yargs types seem wrong; it's ok for builder to be a function
```

Comment:
When I looked at this code locally, that wasn't actually what it was complaining about (being a function). What's the exact error TS is giving you here?

Reply:
Must have been the ESM experiment I did yesterday that changed the TS error for me. I'm seeing the same thing now 👍